dev_requirements.txt

```diff
@@ -13,4 +13,5 @@ pytest-cov==2.10.1
 sphinx>=5
 sphinx-autodoc-typehints==1.11.1
 jinja2<3.1.0
-sphinx_rtd_theme==0.5.1
\ No newline at end of file
+sphinx_rtd_theme==0.5.1
+pydantic-settings>=2.0.0
```

Comment:
this should be moved to the actual requirements as it's needed in the package at runtime